### PR TITLE
Apigee hybrid on GKE

### PR DESCRIPTION
**Pull request by apichick**

### CODE REVIEW

File: blueprints/apigee/hybrid-gke/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Apigee Hybrid on GKE
+
+This example installs Apigee hybrid a non-prod environment on a GKE private cluster using Terraform and ansible.
+The terraform configuration deploys all the required infrastructure including a management VM that used to run an ansible playbook to the the actual Apigee Hybrid setup.
```
[the remark this answers is not in the document]
Comment (ludoo) [reply blueprints/apigee/hybrid-gke/README.md:4]: management VM used to run

**Review by ludoo — COMMENTED** [review]
What is inside the zip file?

File: blueprints/apigee/hybrid-gke/apigee.tf [hunk 2]
```diff
@@ -0,0 +1,46 @@
+/**
```
Comment (juliocc) [inline blueprints/apigee/hybrid-gke/apigee.tf:1]: use `#` comments for the header

File: blueprints/apigee/hybrid-gke/variables.tf [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+ * limitations under the License.
+ */
+
+variable "cluster_machine_type" {
+  description = "Cluster nachine type."
+  type        = string
+  default     = "e2-standard-4"
+}
+
+variable "cluster_subnet_cidr_block" {
```
[the remark this answers is not in the document]
Comment (apichick) [reply blueprints/apigee/hybrid-gke/variables.tf]: I've put all cluster related network configs in a variable

**Comment by apichick** [comment]
All changes done

**Comment by ludoo** [comment]
> What is inside the zip file?

Miren, what is in the zip?

**Pull request merged**